types/globals.d.ts

```diff
@@ -1,6 +1,22 @@
 declare global {
 	var REMOTE_DATA_BLOCKS: LocalizedBlockData | undefined;
 	var REMOTE_DATA_BLOCKS_SETTINGS: LocalizedSettingsData | undefined;
+
+	/**
+	 * Generic Tracks properties sent from MU Plugins.
+	 */
+	var VIP_TRACKS_BASE_PROPS: VipTracksBaseProps | undefined;
+}
+
+export interface VipTracksBaseProps {
```

Comment:
Put this in a separate file like `tracks.d.ts`

Reply:
https://github.com/Automattic/remote-data-blocks/pull/185/commits/be36ae58a7d2ef1d97a1d4591a620de7ac137190